README.md

```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
 
 All versions of the site that relate to past and future versions will be named after their Kubernetes release number. For example, [the old branch for the 1.1 docs is called `release-1.1`](https://github.com/kubernetes/kubernetes.github.io/tree/release-1.1).
 
+## Partners {#partners}
+
+For Kubernetes partners, once verified by the community manager, can get their logos added to the partner section on [community page](http://k8s.io/community). The partner solutions should have a landing page / guide / blog at their end that talks about the integration details with kubernetes.
```

Comment:
Partners can get their logos added to the partner section on [community page](http://k8s.io/community) by following the below steps. Partners will need to have a URL that is specific to integrating with Kubernetes; this URL will be the destination when the logo is clicked.